src/test/java/codeu/model/store/basic/ActivityStoreTest.java

```diff
@@ -112,6 +112,6 @@ public void testGetActivities_badRequest() {
     private void assertEquals(Activity expectedActivity, Activity actualActivity) {
```

Comment:
Consider implementing `equals` method in the Activity class. That way, you can just use:
```
Assert.assertEquals(activityA, activityB);
```